shoop/campaigns/admin_module/form_sets.py

```diff
@@ -89,7 +102,7 @@ class CatalogEffectsFormSet(BaseFormset):
     model = ProductDiscountEffect
 
     def get_queryset(self):
-        return self.owner.effects.instance_of(self._get_actual_model())
+        return self.owner.product_effects.instance_of(self._get_actual_model())
```

Comment:
This also can be derived from the `EffectsFormset` defined in this comment: https://github.com/shuup/shuup/pull/561/commits/3fbbc88ccbf8af7f3b36e475f8c5d040a1427e76#r68250882